### ISSUE TITLE
Daemon handling

### ISSUE THREAD

**Issue by Farthen** [wurstmineberg/wurstminebot#32]
At the moment the bot does have a daemonize option but it is poorly written and horrible. I would suggest to remove it for 4.0 and just let systemd or similar daemon runtime monitoring tools handle this. No need to duplicate things badly here.

**Comment by fenhl** [comment]
Agreed. Since we are going to use Python 3.4 features anyway (which on our server means waiting for the Debian jessie release), moving to systemd is acceptable. Similarly, the goal is to split [init-minecraft](https://github.com/wurstmineberg/init-minecraft) into the `minecraft` and `wmb` Python libs and a clean systemd init script.
